solpolpy/plotting.py

```diff
@@ -141,3 +160,81 @@ def get_colormap_str(meta: fits.Header) -> str:
         color_map = "soholasco2"
 
     return color_map
+
+
+
+def generate_rgb_image(collection,
+                        inner_radius=1,
+                        outer_radius=32,
+                        mask_radius=6,
+                        display=None,
+                        save_path=None,
+                        enhancement_method='nrgf',
+                        return_outputs=False):
+    """
+    Generate an RGB color image from an NDCollection based on Patel et al. 2023 Res. Notes AAS 7 241.
+
+    Parameters:
+        collection (NDCollection): A collection of NDCube objects containing solar data.
+        inner_radius (float, optional): Inner radius for the radial bin edges in solar radii. Default is 1.
+        outer_radius (float, optional): Outer radius for the radial bin edges in solar radii. Default is 32.
+        mask_radius (float, optional): Application radius for the enhancement mask in solar radii. Default is 6.
+        display (bool, optional): If True, display the generated color image. Default is None.
+        save_path (str, optional): File path to save the image. Default is None.
+        enhancement_method (str, optional): The radial enhancement method to use. Can be 'nrgf', 'fnrgf', 'rhef',
+                or 'none'. Default is 'nrgf'.
+        return_outputs (bool, optional): If True, returns the processed NDCollection object. Default is False.
+
+    Returns:
+        np.ndarray: Generated color image with RGB channels.
+    """
```

Comment:
Please reformat the docstring to comply with RST so that it renders properly in the docs.